sandboxed-containers-operator/instance/base/aws/kustomization.yaml

```diff
@@ -1,8 +1,7 @@
----
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
```

Comment:
Was this intended to be inside the base folder?

Reply:
I see a little bit better what you are trying to do here.  I wonder if this might be a good use for a component instead of putting it in a sub folder in the base?

I have been using components in the gitops instance and it has been a nice we to add some composable features that can be easily consumed by an overlay.

Reply:
@strangiato Possible. I saw a few ways to do it. Let's do a working session to decide what will be most elegant.